op-challenger/game/fault/test/game_builder.go

```diff
@@ -112,22 +114,22 @@ func (s *GameBuilderSeq) ExpectStepAttack() *GameBuilderSeq {
 		Type:       types.ActionTypeStep,
 		ParentIdx:  s.lastClaim.ContractIndex,
 		IsAttack:   true,
-		PreState:   s.builder.CorrectPreState(traceIdx),
-		ProofData:  s.builder.CorrectProofData(traceIdx),
-		OracleData: s.builder.CorrectOracleData(traceIdx),
+		PreState:   s.builder.CorrectPreState(*traceIdx),
+		ProofData:  s.builder.CorrectProofData(*traceIdx),
+		OracleData: s.builder.CorrectOracleData(*traceIdx),
```

Comment:
Not sure this makes sense - seems like the `ClaimBuilder` API should just use `*big.Int`. The `big.Int` docs specifically say:
> shallow copies of Ints are not supported and may lead to errors.

so we always need to use a pointer.